README.md

```diff
@@ -1,25 +1,34 @@
+# KVDB
+
 This project aims to implement a KV database.
 
 <details>
 <summary>Dependencies</summary>
 
 * meson
 * ninja
-* fmtlib  @ https://github.com/fmtlib/fmt
-* tl::expected @ https://github.com/TartanLlama/expected
-* gtest
+* [fmtlib](https://github.com/fmtlib/fmt)
+* [tl::expected](https://github.com/TartanLlama/expected)
+* [gtest](https://github.com/google/googletest)
 
 These are all external dependencies. Please install them on your platform by
 yourself.
 </details>
 
-# Build
+## Build
+
+```bash
+meson setup --native-file=native.ini build/     # generate build instructions
+meson compile -C build                          # build the project
+```
+
+## Run
+
 ```bash
-meson build     # generate build instructions
-ninja -C build  # compile the project
-./build/kv      # run the executable
+./build/kv              # run the executable
+meson test -C build     # run tests
 ```
 
+## References
 
-# References:
-https://github.com/balloonwj/CppGuide/blob/master/articles/leveldb源码分析/README.md
+1. https://github.com/balloonwj/CppGuide/blob/master/articles/leveldb源码分析/README.md
```

Comment:
Codacy has a fix for the issue: [Bare URL used](https://app.codacy.com/gh/Congee/kvdb/pullRequest?prid=10017121)
```suggestion
1. <https://github.com/balloonwj/CppGuide/blob/master/articles/leveldb源码分析/README.md>
```